redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/RedbeamsStartEvent.java

```diff
@@ -2,12 +2,14 @@
 
 import com.sequenceiq.flow.core.FlowEvent;
 import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertReplaceInRedBeamsSuccess;
 import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerFailed;
 import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerSuccess;
 
 public enum RedbeamsStartEvent implements FlowEvent {
 
-    REDBEAMS_START_EVENT(),
+    REDBEAMS_CERT_REPLACE_EVENT(),
```

Comment:
* `REPLACE` -> `ROTATE`? It would express the goal more appropriately.
* This shall likely not be the init event; I do not think it is possible to change the root cert of a stopped AWS RDS instance, and the same goes for GCP Cloud SQL. Even if "rotation on cloud provider side" is not implemented at the moment, the rotation step shall conceptually take place after the DB server has been started.